www/js/Widget.js

```diff
@@ -215,11 +215,18 @@ class Widget {
 
   on (eve, callback) {
     if (!this.events[eve]) {
-      return console.error(`Widget: ${eve} is not a Widget event`)
+      // return console.error(`Widget: ${eve} is not a Widget event`)
+      this.events[eve] = []
     }
     this.events[eve].push(callback)
   }
 
+  emit (eve, data) {
+    this.events[eve].forEach(i => {
+      i(data)
```

Comment:
this looks like it should technically work, but we should probably change "i" to something else for readability sake (ie. when someone looks at this code down the line). we probably want something more descriptive like "func" or "callback" (it's also convention to abbreviate callback functions to "cb") as opposed to "i" which is conventionally reserved for "iterator" vaiables (ie. a number/index)

Reply:
also, don't forget to update the "Usage" section in the documentation comment at the top of the file to include a mention of the new .emit() method

Reply:
what example event should i use for the usage section?
`// to fire all registered callbacks of an event
  w.emit('click')`

Reply:
yep